package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "saf-lambda-function",
+  "description": "Serverless lambda function for the [SAF CLI](https://github.com/mitre/saf)",
+  "version": "0.0.1",
+  "private": true,
+  "dependencies": {
+    "@mitre/saf": "latest",
+    "winston": "^3.6.0",
+    "winston-daily-rotate-file": "^4.6.1"
+  },
+  "scripts": {
+    "make-event": "serverless generate-event -t aws:s3 > test/event.json",
+    "test": "serverless invoke local --function saf-lambda-function --path test/event.json",
+    "check": "serverless deploy list"
```

Comment:
Are these three scripts helpful or confusing? Check out the readme for the documentation in the testing section.

Reply:
You aren't referencing the 'check' command at least at the version of the readme i'm seeing.

These other two scripts are fine.  I would emphasize within the readme that these are optional commands one can run as part of their testing process - it should be emphasized that they are not the only means through which one should be testing things.